dask/array/fft.py

```diff
@@ -10,6 +11,9 @@
 except ImportError:
     scipy = None
 
+from dask.array.creation import linspace as _linspace
+from dask.array.core import concatenate as _concatenate
```

Comment:
We use absolute imports consistently throughout. I would prefer you continue that convention here.